src/services.js

```diff
@@ -18,15 +18,15 @@ export const getCurrentUser = async () => {
   // write your code here
 };
 
-export const updateCourse = async () => {
+export const updateCourseService = async () => {
   // write your code here
 };
 
 export const logout = async () => {
   // write your code here
 };
 
-export const deleteCourse = async () => {
+export const deleteCourseService = async () => {
```

Comment:
Why did you add `Service` word here and did not add for other services?

Reply:
because  of deleteCourse slice and deleteCourse service names duplication